.yarnrc.yml

```diff
@@ -1,3 +1,9 @@
-yarnPath: ".yarn/releases/yarn-berry.cjs"
-nodeLinker: "node-modules"
 enableGlobalCache: true
+
+nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
+    spec: "@yarnpkg/plugin-workspace-tools"
```

Comment:
What does the plugin do? Is it to support the `yarn workspace foreach` directive? 

Can you add a comment/URL-link back tot he plugin documentation?